### ISSUE TITLE
Cannot launch debug after User Update

### ISSUE THREAD

**Issue by Dgiulian** [microsoft/vscode#56680]
Issue Type: <b>Bug</b>

After I upgraded to User Mode of VSCode, and use it as a regular user (not Admin) I cannot launch the debugger. But if I start it as Admin or use the previous version, everything works as expected. FYI I'm using it behind a proxy.
The error I get is: 
Cannot connect to runtime process, timeout after 10000ms - (reason: Cannot connect to the target: connect ERRCONNREFUSED 127.0.0.1:9222).

Here is my launch.json
{
     "type": "chrome",
     "request": "launch",
     "name": "Launch Angular App in chrome",
     "url": "http://localhost:4200",
     "webRoot": "${workspaceFolder}/admin"
 },

VS Code version: Code 1.26.1 (493869ee8e8a846b0855873886fc79d480d342de, 2018-08-16T18:38:57.434Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD A10-8700P Radeon R6, 10 Compute Cores 4C+6G (4 x 1797)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.45GB (8.42GB free)|
|Process Argv|C:\Users\giuli\AppData\Local\Programs\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (16)</summary>

Extension|Author (truncated)|Version
---|---|---
ng-template|Ang|0.1.9
path-intellisense|chr|1.4.2
vscode-eslint|dba|1.4.12
githistory|don|0.4.2
EditorConfig|Edi|0.12.4
tslint|eg2|1.0.35
vscode-npm-script|eg2|0.3.5
prettier-vscode|esb|1.6.1
angular-essentials|joh|0.4.0
Angular2|joh|6.1.5
vscode-language-pack-es|MS-|1.26.3
debugger-for-chrome|msj|4.8.2
angular2-inline|nat|0.0.17
material-icon-theme|PKi|3.5.2
git-merger|sha|0.4.1
vscode-todo-highlight|way|1.0.4

(1 theme extensions excluded)

</details>

**Comment by Dgiulian** [comment]
After digging arround I found out I couldn't open chrome from the integrated terminal as a normal user (Permision denied). So VS Code can't open it with debug flag either.
I understand now that this isn't technically a VSCode bug, but I feel that error message should be a little more helpful.  Especially if the new version encourages one to use the User version

**Comment by roblourens** [comment]
How do you normally launch Chrome? As admin?

Do you have some unusual setup that caused this?

**Comment by CraigHead** [comment]
If it's helpful (or not) .. I get the same error when I try to debug my Electron apps after the 1.27.2 (user) update. My config is as follows. 
`
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Launch",
            "type": "chrome",
            "request": "launch",
            "runtimeExecutable": "${workspaceRoot}/node_modules/electron/dist/electron.exe",
            "windows": {
                "runtimeExecutable": "${workspaceRoot}/node_modules/electron/dist/electron.exe"
            },
            "runtimeArgs": [
                "${workspaceRoot}/main.js",
                "--remote-debugging-port=9222"
            ],
            "webRoot": "${workspaceRoot}",
            "sourceMaps": true,
            "outFiles": ["${workspaceFolder}/**/*.js"]
        }
    ]
}
`

**Comment by CraigHead** [comment]
Great, so in an effort to help understand this issue today I force removed my git repo for this electron project. Re-cloned and re-installed npm packages. Now I can't reproduce this issue. Leaving me to think it was either environmental to that repro OR there was a npm package that was throwing an error and has since then been fixed. I wasn't able to compare the broken directory because I had tried too many things that made the broken directory diverge enough that my comparison was invalid. (e.g. I tried changing typescript versions, tslint rules, etc). So hopefully someone else with the same problem (e.g. @Dgiulian) can offer additional details.
